domains/mirjalol.is-a.dev.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "mirjalol2008",
+    "email": "uztelecomtv072@gmail.com"
+  },
+  "record": {
+    "CNAME": "index-five-kohl.vercel.app"
```

Comment:
Little content provided on site. Reopen this PR when you have completed your site.